root/admin/attributes/Language.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * @flow
+ * eslint-disable flowtype/no-mutable-array
+ * Copyright (C) 2019 Anirudh Jain
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import Layout from '../../layout';
+import {compare} from '../../static/scripts/common/i18n';
+
+type Props = {
+  attributes: Array<LanguageT>,
+  model: string,
+};
+
+const Language = ({model, attributes}: Props) => {
+  return (
+    <Layout fullWidth title={model ? model : l('Language')}>
+      <h1>
+        <a href="/admin/attributes">{l('Attributes')}</a>
+        <text>{' / ' + l('Language')}</text>
+      </h1>
+      <table className="tbl">
+        <thead>
+          <tr>
+            <th>{l('ID')}</th>
+            <th>{l('Name')}</th>
+            <th>{l('ISO 639-1')}</th>
+            <th>{l('ISO 639-2/B')}</th>
+            <th>{l('ISO 639-2/T')}</th>
+            <th>{l('ISO 639-3')}</th>
+            <th>{l('Frequency')}</th>
+            <th>{l('Actions')}</th>
+          </tr>
+        </thead>
+        {attributes
+          .sort((a, b) => compare(a.name.toLowerCase(), b.name.toLowerCase()))
+          .reverse()
+          .sort((a, b) => compare(str(a.frequency), str(b.frequency)))
+          .reverse()
```

Comment:
`.toLowerCase` calls are unneeded here too.

Using `i18n.compare` (for localized string array) to sort attributes by frequency (of type integer) is a bit expensive (and `str` is not defined), I would suggest using `terable/sortBy` instead, see [SidebarLicenses.js](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-01-22/root/layout/components/sidebar/SidebarLicenses.js) for example.

Reply:
`terable/sortBy` might be tricky to work with here.
I push 3 commits to use `lodash/orderBy` instead.
Feel free to reuse it.

Reply:
You can already sort by multiple criteria in a `.sort()` callback. e.g.

```JavaScript
attributes.sort((a, b) => (b.frequency - a.frequency) || compare(a.name, b.name))
```

Will sort by frequency numerically (descending), then by name (ascending - if you want descending there simply swap the argument order).

The `.sort()` callback only needs to return a negative number if `a < b`, or a positive number if `a > b` (0 if they're equal). If `(b.frequency - a.frequency)` is 0 then they have the same frequency, and it'll short-circuit to the `compare` since 0 is falsey.

Reply:
I used lodash on recommendation by yvanzo in the irc, I'd love to discuss about this